socorro/external/es/index_cleaner.py

```diff
@@ -68,13 +74,23 @@ def delete_old_indices(self):
                 # This index doesn't look like a crash index, let's skip it.
                 continue
 
-            # This won't take the week part of our indices into account...
-            index_date = datetime.datetime.strptime(
-                index,
-                self.config.elasticsearch.elasticsearch_index
-            )
-            # So we need to get that differently, and then add it to the date.
-            index_date += datetime.timedelta(weeks=int(index[-2:]))
+            if predicate is None or predicate(index):
+                index_client.delete(index)
```

Comment:
In the case where we've run this from the command line (as opposed to crontabber), I'd really like it to tell me what it deleted. One way to do that is pass in a `verbose` flag and then have it spit it out to the logger. Another way is have this method return the list of indexes it deleted and then the caller can print them out.